.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "fxa/server",
+  "rules": {
+    "semi": 0,
+    "strict": 0
+  }
```

Comment:
You can remove these two rules. As of [**eslint-config-fxa@1.6.0**](https://github.com/pdehaan/eslint-config-fxa/blob/aba8a0a09c16a0b460bd4b079e04d33207fb8877/server.js) the "fxa/server" rules use:

```
  rules: {
    'curly': 0,
    'semi': [2, 'never']
  }
```

And `strict: 0` should be the default.